src/main/resources/assets/gregtech/lang/zh_cn.lang

```diff
@@ -349,762 +351,770 @@ metaitem.energy.module.tooltip=可充电电池
+metaitem.electric.piston.opv.name=电力活塞（OpV）
+
+metaitem.robot.arm.lv.name=机械臂（LV）
+metaitem.robot.arm.lv.tooltip=按指定速率转运物品，每秒至多可转运 8 件物品（作覆盖板时）
+metaitem.robot.arm.mv.name=机械臂（MV）
+metaitem.robot.arm.mv.tooltip=按指定速率转运物品，每秒至多可转运 32 件物品（作覆盖板时）
+metaitem.robot.arm.hv.name=机械臂（HV）
+metaitem.robot.arm.hv.tooltip=按指定速率转运物品，每秒至多可转运 64 件物品（作覆盖板时）
+metaitem.robot.arm.ev.name=机械臂（EV）
+metaitem.robot.arm.ev.tooltip=按指定速率转运物品，每秒至多可转运 3 组物品（作覆盖板时）
+metaitem.robot.arm.iv.name=机械臂（IV）
+metaitem.robot.arm.iv.tooltip=按指定速率转运物品，每秒至多可转运 8 组物品（作覆盖板时）
+metaitem.robot.arm.luv.name=机械臂（LuV）
+metaitem.robot.arm.luv.tooltip=按指定速率转运物品，每秒至多可转运 16 组物品（作覆盖板时）
+metaitem.robot.arm.zpm.name=机械臂（ZPM）
+metaitem.robot.arm.zpm.tooltip=按指定速率转运物品，每秒至多可转运 16 组物品（作覆盖板时）
+metaitem.robot.arm.uv.name=机械臂（UV）
+metaitem.robot.arm.uv.tooltip=按指定速率转运物品，每秒至多可转运 16 组物品（作覆盖板时）
+metaitem.robot.arm.uhv.name=机械臂（UHV）
+metaitem.robot.arm.uhv.tooltip=按指定速率转运物品，每秒至多可转运 16 组物品（作覆盖板时）
+metaitem.robot.arm.uev.name=机械臂（UEV）
+metaitem.robot.arm.uev.tooltip=按指定速率转运物品，每秒至多可转运 16 组物品（作覆盖板时）
+metaitem.robot.arm.uiv.name=机械臂（UIV）
+metaitem.robot.arm.uiv.tooltip=按指定速率转运物品，每秒至多可转运 16 组物品（作覆盖板时）
+metaitem.robot.arm.uxv.name=机械臂（UXV）
+metaitem.robot.arm.uxv.tooltip=按指定速率转运物品，每秒至多可转运 16 组物品（作覆盖板时）
+metaitem.robot.arm.opv.name=机械臂（OpV）
+metaitem.robot.arm.opv.tooltip=按指定速率转运物品，每秒至多可转运 16 组物品（作覆盖板时）
+
+metaitem.field.generator.lv.name=力场发生器（LV）
+metaitem.field.generator.mv.name=力场发生器（MV）
+metaitem.field.generator.hv.name=力场发生器（HV）
+metaitem.field.generator.ev.name=力场发生器（EV）
+metaitem.field.generator.iv.name=力场发生器（IV）
+metaitem.field.generator.luv.name=力场发生器（LuV）
+metaitem.field.generator.zpm.name=力场发生器（ZPM）
+metaitem.field.generator.uv.name=力场发生器（UV）
+metaitem.field.generator.uhv.name=力场发生器（UHV）
+metaitem.field.generator.uev.name=力场发生器（UEV）
+metaitem.field.generator.uiv.name=力场发生器（UIV）
+metaitem.field.generator.uxv.name=力场发生器（UXV）
+metaitem.field.generator.opv.name=力场发生器（OpV）
+
+metaitem.emitter.lv.name=发射器（LV）
+metaitem.emitter.mv.name=发射器（MV）
+metaitem.emitter.hv.name=发射器（HV）
+metaitem.emitter.ev.name=发射器（EV）
+metaitem.emitter.iv.name=发射器（IV）
+metaitem.emitter.luv.name=发射器（LuV）
+metaitem.emitter.zpm.name=发射器（ZPM）
+metaitem.emitter.uv.name=发射器（UV）
+metaitem.emitter.uhv.name=发射器（UHV）
+metaitem.emitter.uev.name=发射器（UEV）
+metaitem.emitter.uiv.name=发射器（UIV）
+metaitem.emitter.uxv.name=发射器（UXV）
+metaitem.emitter.opv.name=发射器（OpV）
+
+metaitem.sensor.lv.name=传感器（LV）
+metaitem.sensor.mv.name=传感器（MV）
+metaitem.sensor.hv.name=传感器（HV）
+metaitem.sensor.ev.name=传感器（EV）
+metaitem.sensor.iv.name=传感器（IV）
+metaitem.sensor.luv.name=传感器（LuV）
+metaitem.sensor.zpm.name=传感器（ZPM）
+metaitem.sensor.uv.name=传感器（UV）
+metaitem.sensor.uhv.name=传感器（UHV）
+metaitem.sensor.uev.name=传感器（UEV）
+metaitem.sensor.uiv.name=传感器（UIV）
+metaitem.sensor.uxv.name=传感器（UXV）
+metaitem.sensor.opv.name=传感器（OpV）
 
 metaitem.tool.datastick.name=闪存
 metaitem.tool.datastick.tooltip=一个小容量的数据存储器
 metaitem.tool.dataorb.name=数据球
 metaitem.tool.dataorb.tooltip=一个大容量的数据存储器
```

Comment:
i tend to make tooltips informally explanatory, let alone en_us be keeping that way too

Reply:
~~how do you explain when there's a stack of data stick and they're still called "**a** storage" in the tooltip, while en_us has the same problem~~
英语有冠词，所以前面非有 a 不可，但是汉语没有。尤其此处不强调它是**一个**存储器

Reply:
With "a", it's a guy explaining what it is to you. Without it, it's a brand, a title, a term, so you can never see a label beside a dusty ancient ware saying that it is

**_A BOWL_**

but like Bowl (Han Dynasty). It's without "a', and it takes the job of what an item's displayed name does, but tooltips shouldn't be that serious. It's the mod creator wants you to actually know what it is. Let's have another example.

**Thunder Strike**
_An almighty sword that summons thunderstorm._

instead of 一把能够召唤雷暴的全能利剑, if you translate it to 全能雷暴召唤剑 (Almighty Thunderstorm Summoner), that's deadass weird. That feels like there are literally TWO names inside one single tooltip box. Of course it's legit in some of the cases tho, like

**Skeleton**
_Arrow-shooting Crop Growth Booster 9000_

but that's to make you have a good laugh over that.

I truly apologize but i finna keep it. iykyk.